grails-app/services/org/pih/warehouse/core/LocationService.groovy

```diff
@@ -343,6 +352,10 @@ class LocationService {
         return locations.values().toList()
     }
 
+    List getSubmitRequestLocations() {
```

Comment:
Let's call this getRequestingLocations()

Reply:
ok